src/gatsby/package-lock.json

```diff
@@ -5454,9 +5454,9 @@
       }
     },
     "node_modules/@types/react": {
-      "version": "18.2.6",
-      "resolved": "https://registry.npmjs.org/@types/react/-/react-18.2.6.tgz",
-      "integrity": "sha512-wRZClXn//zxCFW+ye/D2qY65UsYP1Fpex2YXorHc8awoNamkMZSvBxwxdYVInsHOZZd2Ppq8isnSzJL5Mpf8OA==",
+      "version": "17.0.60",
+      "resolved": "https://registry.npmjs.org/@types/react/-/react-17.0.60.tgz",
+      "integrity": "sha512-pCH7bqWIfzHs3D+PDs3O/COCQJka+Kcw3RnO9rFA2zalqoXg7cNjJDh6mZ7oRtY1wmY4LVwDdAbA1F7Z8tv3BQ==",
```

Comment:
What was this change? Just curiosity 🙂

Reply:
I'm not sure exactly, but I have been using yarn instead of npm and I think I had the wrong version